### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by RyanTamulevicz** [saleor/saleor#9690] [Ideas]
I know I and I am sure many other people enjoy using discord for finding people to help and learn from. I would love a discord channel for Saleor and I think it would help create an even bigger community for you guys!

This would also help the discussion about the "smaller" Saleor projects like the storefront, dashboard, checkout etc. where the discussion are a lot less used in.

**Comment by karolkielecki** [comment]
Hi @RyanTamulevicz, 
That's a great idea, and we've been thinking of creating a Discord channel for a while. These days are pretty busy for us, but we'll get back to the topic soon!

**Reply by janak819** [reply]
is it available now? if so pls send me a link

**Reply by peelar** [reply]
It is not @janak819, no updates on that. If you have any issues/questions, please submit them on GitHub.

**Comment by RyanTamulevicz** [comment]
@karolkielecki That would be great! I know I and a bunch of others would love that! Thanks for you guys work on Saleor though!

**Comment by waveywaves** [comment]
It would be a great idea to have a discord channel for Saleor ! :D 
+1

**Comment by kk2211** [comment]
+1
Would love to have a Discord or Slack channel. Would make communication with community members way easier

**Reply by karolkielecki** [reply]
Hi there! I know you and other community members would like to use Discord. However, we would like to focus on GitHub for now. We believe using GitHub offers much more -- more transparency, more effective in terms of fixing bugs, discussing feedback, and finding solutions for specific use cases. There is a higher chance that someone new to Saleor will find a solution or similar issue on GitHub rather than on Discord. 

But, it doesn't mean we won't use Discord. We'd like to use it but in the future.

**Reply by kk2211** [reply]
Agreed with Github being more efficient for bug fixing and transparency

However the main benefit of Discord(or slack) is the ease with which the community can participate and provide feedback to different queries.
I feel that Github Discussions would only be opened/viewed by people who are currently facing/resolving issues, whereas with Discord much wider audience can easily participate/view discussions.

**Comment by patrys** [comment]
Folks, it only took us a year, but here it is: https://discord.com/invite/H52JTZAtSH
